thrust/benchmarks/bench/find_if/basic.cu

```diff
@@ -0,0 +1,36 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <thrust/detail/internal_functional.h>
+#include <thrust/device_vector.h>
+#include <thrust/fill.h>
+#include <thrust/find.h>
+
+#include "nvbench_helper.cuh"
+
+template <typename T>
+void find_if(nvbench::state& state, nvbench::type_list<T>)
+{
+  T val = 1;
+  // set up input
+  const auto elements       = static_cast<std::size_t>(state.get_int64("Elements"));
+  const auto common_prefix  = state.get_float64("MismatchAt");
+  const auto mismatch_point = static_cast<std::size_t>(elements * common_prefix);
+
+  thrust::device_vector<T> dinput(elements, thrust::no_init);
+  thrust::fill(dinput.begin(), dinput.begin() + mismatch_point, T{0});
+  thrust::fill(dinput.begin() + mismatch_point, dinput.end(), val);
+
+  state.add_global_memory_reads<T>(mismatch_point);
```

Comment:
Thinking about it, if you scan `dinput` contiguously from the beginning, you need to read `mismatch_point` many zeros, and at least one more element to detect the mismatch. So it shoul dbe:
```suggestion
  state.add_global_memory_reads<T>(mismatch_point + 1);
```